### PR TITLE
Update dpm-advice-basic-melee.txt

### PR DESCRIPTION
**Pull request by Sky-Justice**
Minor Changes: 
- Fixed bullet point formatting error under weapon breakdown. 
- Cleaned some redundant phrases
- Added a missing Dragon Rider Lance emoji to weapon breakdown

# PvME Change Submission
Thank you for helping maintain our resources! Below is a quick sanity checklist that helps make sure we can keep track of what is changing. Please fill it out :)

## Sanity Checks
- [x] - The title of this pull request clearly describes the change I would like to make.
- [x] - If there are multiple changes in this pull request, they are all related.
- [x] - I have tried to write clear a commit message(s) that describes the changes I made.

### CODE REVIEW

File: dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt [hunk 1]
```diff
@@ -77,15 +77,15 @@ Melee thresholds are stronger than basics however are more taxing to use. These
...
-⬥ __**Power Thresholds:**__
-    • *Examples:* <:assault:535532853979512842> <:destroy:535532879330148352> <:cane:535532878969438210> <:bloodtendrils:535532854327640064> <:slaughter:535532879237873666>
+⬥ __**Power:**__
+    • <:assault:535532853979512842> <:destroy:535532879330148352> <:cane:535532878969438210> <:bloodtendrils:535532854327640064> <:slaughter:535532879237873666>
         - <:destroy:535532879330148352> *does stun however is more renowned for its damage.*
 
-⬥ __**Utility Thresholds:**__
-    • *Examples:* <:fbackhand:535532879346794516> <:gflurry:535532879283879977> <:quake:535532879506309150>
+⬥ __**Utility:**__
+    •  <:fbackhand:535532879346794516> <:gflurry:535532879283879977> <:quake:535532879506309150>
```
Comment (nullopt) [inline dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt:85]: Has two spaces after bullet

File: dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt [hunk 2]
```diff
@@ -128,24 +128,24 @@ Melee Ultimates deal maximum damage and or elevate other abilities. They typical
...
-    • **__Damage__**: Auto attack damage from the respective weapon.
+    \• **__Damage__**: Auto attack damage from the respective weapon.
         - <:darkiceshard:915232154160594974> has a t88 damage value of: 844.5.
         - <:darkicesliver:915232153787334677> has an off-hand t88 damage value of: 422.4.
         - <:dragonriderlance:643161541931171872> has a t80 damage value of 1788.
 
-    • **__Attack Speed__**: Auto attack speed of each weapon.
+    \• **__Attack Speed__**: Auto attack speed of each weapon.
         - <:darkiceshard:915232154160594974> <:darkicesliver:915232153787334677> each auto attack once every 2.4s (4t)
         - <:dragonriderlance:643161541931171872> auto attacks once every 3.6s (6t)
```
Comment (nullopt) [inline dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt:143]: Needs consistent '.' usage at end of lines

File: dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt [hunk 3]
```diff
@@ -42,20 +42,20 @@ Melee basics are common use abilities with 2 purposes:
...
-    • *Example: <:bd:535532854281764884> <:dive:1049378668197195808> <:kick:535532879132885025> <:backhand:535532854302605333>*
+⬥ **__Utility:__**
+    • <:bd:535532854281764884> <:dive:1049378668197195808> <:kick:535532879132885025> <:backhand:535532854302605333>
 
-⬥ __**Subpar Basics**__
-    • *Example:* <:slice:535532879594258432> <:Sacrifice:513201065907322880> <:Tuskas:513201065513058306>
+⬥ __**Subpar:**__
+    • <:slice:535532879594258432> <:Sacrifice:513201065907322880> <:Tuskas:513201065513058306>
 
 **__Sample Auxiliary Comboes__**
```
Comment (nullopt) [inline dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt:60]: Comboes -> Combos according to Cambridge Dictionary

File: dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt [hunk 4]
```diff
@@ -77,15 +77,15 @@ Melee thresholds are stronger than basics however are more taxing to use. These
 ⬥ Sacrifice adrenaline (-15%)
 ⬥ Deal higher damage
```
Comment (nullopt) [inline dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt:78]: Consistent '.' usage

File: dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt [hunk 5]
```diff
@@ -100,17 +100,17 @@ Melee Ultimates deal maximum damage and or elevate other abilities. They typical
 ⬥ Deal devastating damage
 ⬥ Provide a longstanding buff
```
Comment (nullopt) [inline dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt:101]: Consistent '.' usage

**Review by nullopt — CHANGES_REQUESTED** [review]

File: dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt [hunk 6]
```diff
@@ -75,17 +75,17 @@ __**Utility Examples:**__
 .
 Melee thresholds are stronger than basics however are more taxing to use. These abilities have 2 purposes:
 ⬥ Sacrifice adrenaline (-15%)
```
Comment (nullopt) [inline dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt:77]: Missing a `.` here

File: dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt (hunk 2 above)
Comment (nullopt) [inline dpm-advice/dpm-advice-basic/dpm-advice-basic-melee.txt:143]: And on these two lines here

**Review by nullopt — CHANGES_REQUESTED** [review]

**Review by Sky-Justice — COMMENTED** [review]
Formatting changes for dpm-advice-basic-melee

**Comment by Ferroric** [comment]
Resolved all conflicts, may need to revisit after merging for any other small issues

**Pull request merged**